File: swea/D3/3431.준환이의 운동관리/3431.cpp
#include <iostream>

using namespace std;

int T;

int main(){
  cin >> T;
  for(int t = 1; t <= T; t++){
    int l, u, x;
    cin >> l >> u >> x;

    int ans = 0;
    if(x < l) ans = l - x;
    else if(x > u) ans = -1;
    else ans = 0;

    cout << '#' << t << ' ' << ans << endl;
  }
}

File: swea/D3/3431.준환이의 운동관리/3431.py
T = int(input())
for tc in range(1, T+1):
    l, u, x = map(int, input().split())

    ans = 0
    if x < l:
        ans = l - x
    elif x > u:
        ans = -1
    else:
        ans = 0

    print(f'#{tc} {ans}')